src/test/java/com/ericsson/ei/test/utils/TestConfigs.java

```diff
@@ -122,4 +108,30 @@ protected void setRules() {
         System.setProperty("rules", " /rules/ArtifactRules-Eiffel-Agen-Version.json");
     }
 
+    protected static void setSystemProperties(int port) {
+        System.setProperty("rabbitmq.port", "" + port);
+        System.setProperty("rabbitmq.user", "guest");
+        System.setProperty("rabbitmq.password", "guest");
+        System.setProperty("waitlist.initialDelayResend", "500");
+        System.setProperty("waitlist.fixedRateResend", "100");
+        LOGGER.info("Message bus port:{}", port);
+        LOGGER.debug("Done setting up message bus properties");
```

Comment:
Remove this log message, it doesn't add any value since the one above is more useful

Reply:
done